pkgs/by-name/ze/zed-editor/package.nix

```
@@ -270,8 +275,8 @@ rustPlatform.buildRustPackage rec {
       install -Dm755 $release_target/zed $out/libexec/zed-editor
       install -Dm755 $release_target/cli $out/bin/zeditor
 
-      install -Dm644 ${src}/crates/zed/resources/app-icon@2x.png $out/share/icons/hicolor/1024x1024@2x/apps/zed.png
-      install -Dm644 ${src}/crates/zed/resources/app-icon.png $out/share/icons/hicolor/512x512/apps/zed.png
+      install -Dm644 ${finalAttrs.src}/crates/zed/resources/app-icon@2x.png $out/share/icons/hicolor/1024x1024@2x/apps/zed.png
+      install -Dm644 ${finalAttrs.src}/crates/zed/resources/app-icon.png $out/share/icons/hicolor/512x512/apps/zed.png
```

```suggestion
      install -Dm644 $src/crates/zed/resources/app-icon@2x.png $out/share/icons/hicolor/1024x1024@2x/apps/zed.png
      install -Dm644 $src/crates/zed/resources/app-icon.png $out/share/icons/hicolor/512x512/apps/zed.png
```

Might as well access it through the shell variable

---

True, fixed, thanks!